### ISSUE TITLE
Keyboard navigation wraparound behavior

### ISSUE THREAD

**Issue by evanmiller** [pixelspark/mbtablegrid#19]
I think it's weird that arrow-key navigation wraps around (e.g. Move Left in the first column goes to the last column and up a row, Move Right in the last column goes to the next row and first column)

Excel and Numbers don't do this. If it's okay I'd like to rip this behavior out.

**Comment by evanmiller** [comment]
As a substitute for the wraparound behavior, we could implement:

```
moveToBeginningOfLine:
moveToBeginningOfLineAndModifySelection:
moveToEndOfLine:
moveToEndOfLineAndModifySelection:
```

This will allow jumping to the first or last column via Command-Left, Command-Right, etc. These commands are supported in both Numbers and Excel.

**Comment by evanmiller** [comment]
Fixed in #44
